src/app/pages/address/address.component.html

```diff
@@ -84,12 +84,12 @@
             <div class="ark-address-summary voters-container" style="display: block;" *ngIf="!openVoters" #voters>
                 <div class="voters-button" (click)="showBlock()"></div>
                 <span class="voters-address" *ngFor="let item of addressItem.voters; let i = index;">
-                    <a href="#" (click)="goToAddress($event, item.address)" *ngIf="i < votersNumber">{{item.address}} ({{(item.balance / 100000000) | number: '1.2-2'}} ARK)</a>
+                    <a href="#" (click)="goToAddress($event, item.address)" *ngIf="i < votersNumber">{{item.address}} ({{(item.balance / 100000000) | currencyFormat: 'ARK'}} ARK)</a>
```

Comment:
Need to use a variable to handle this, because we can't manually change it to DEVNET.. the same discussed in #18